airflow/jobs/local_task_job.py

```diff
@@ -204,6 +204,10 @@ def sigusr2_debug_handler(signum, frame):
         finally:
             self.on_kill()
 
+    @staticmethod
+    def get_job_type() -> str:
+        return "LocalTaskJob"
```

Comment:
yes - we can do it this way. It was just doing some step-by-step changes where I got into this place (ans also @uranusjr had the same comment) but yes, I will turn it into class variable, so it might simplify other things as well.

Reply:
Done.